### PR TITLE
Authorization Interceptor for MS

### PR DESCRIPTION
**Pull request by Niveathika92**

### CODE REVIEW

File: components/org.wso2.carbon.msf4j.interceptor.common/pom.xml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
```
Comment (mohanvive) [inline components/org.wso2.carbon.msf4j.interceptor.common/pom.xml]: Please add the license file.

File: components/org.wso2.carbon.msf4j.interceptor.common/src/main/java/org/wso2/carbon/msf4j/interceptor/common/AuthorizationInterceptor.java [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+import java.util.Base64;
+import javax.security.auth.login.LoginContext;
+import javax.security.auth.login.LoginException;
+import javax.ws.rs.core.MediaType;
+
+/**
+ * Interceptor to check basic authorization.
+ */
+public class AuthorizationInterceptor implements RequestInterceptor {
+    private static final Logger log = LoggerFactory.getLogger(AuthorizationInterceptor.class);
```
Comment (mohanvive) [inline components/org.wso2.carbon.msf4j.interceptor.common/src/main/java/org/wso2/carbon/msf4j/interceptor/common/AuthorizationInterceptor.java]: Define the variable 'log' in the upper class.

File: components/org.wso2.carbon.msf4j.interceptor.common/pom.xml (hunk 1 above)
Comment (Niveathika92) [inline components/org.wso2.carbon.msf4j.interceptor.common/pom.xml]: Added license header in 02389cb

File: components/org.wso2.carbon.msf4j.interceptor.common/src/main/java/org/wso2/carbon/msf4j/interceptor/common/AuthorizationInterceptor.java (hunk 2 above)
Comment (Niveathika92) [inline components/org.wso2.carbon.msf4j.interceptor.common/src/main/java/org/wso2/carbon/msf4j/interceptor/common/AuthorizationInterceptor.java]: Changed log variable case in 02389cb

**Pull request merged**